src/Service/AbstractExpressiveNavigationFactory.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Zend\Navigation\Service;
+
+use Psr\Container\ContainerInterface;
+use Traversable;
+use Zend\Config;
+use Zend\Navigation\Exception;
+use Zend\Navigation\Page\ExpressivePage;
+use Zend\Expressive\Router\RouterInterface;
+use Zend\Stdlib\ArrayUtils;
+
+/**
+ * Abstract expressive navigation factory
+ */
+abstract class AbstractExpressiveNavigationFactory
+{
+    /**
+     * @param ContainerInterface $container
+     * @param array              $pages
```

Comment:
Proposed in PSR-5: PHPDoc. ([see example](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#55-examples))